src/App.css

```diff
@@ -1,10 +1,7 @@
 .App-header {
   background-color: #e24e54;
   padding: 20px;
-  color: white;
-}
+  color: white; }
```

Comment:
U no like newlines?

Reply:
thats the generated file ^

Reply:
Let's remove this from the repo and gitignore it